python/lsst/pipe/analysis/fakesAnalysis.py

```diff
@@ -576,16 +579,23 @@ def plotWithOneHist(xs, ys, maskForStats, xLabel, yLabel, title, plotInfoDict):
     sectionSelections = [np.where((xs < xs25))[0], np.where((xs < xs50) & (xs >= xs25))[0],
                          np.where((xs < xs75) & (xs >= xs50))[0], np.where((xs >= xs75))[0]]
 
+    if np.median(meds[:10]) < 0:
+        yArrowPos = medYs - 2.5 * sigmaMadYs
+    else:
+        yArrowPos = medYs + 2.5 * sigmaMadYs
+
+    yScale = (6.0 * sigmaMadYs) / 40.0
+    headLength = 11 * xScale / 20
     for (i, sectionColor) in enumerate(sectionColors):
         if i != len(sectionColors) - 1:
             quartileLine = ax.axvline(sectionBounds[i], color="k", ls="--", zorder=-10, label="Quartiles")
-            ax.arrow(sectionBounds[i], medYs + 2.5 * sigmaMadYs, -1.0 * xScale / 4, 0, head_width=2.0,
-                     head_length=0.25, color=sectionColor, fc=sectionColor)
+            ax.arrow(sectionBounds[i], yArrowPos, -1.0 * xScale / 4, 0, head_width=yScale,
+                     head_length=headLength, color=sectionColor, fc=sectionColor)
         if i != 0:
-            ax.arrow(sectionBounds[i - 1], medYs + 2.5 * sigmaMadYs, xScale / 4, 0, head_width=2.0,
-                     head_length=0.25, color=sectionColor, fc=sectionColor)
+            ax.arrow(sectionBounds[i - 1], yArrowPos, xScale / 4, 0, head_width=yScale,
+                     head_length=headLength, color=sectionColor, fc=sectionColor)
```

Comment:
It took my eyes + brain a while to figure out that the arrow coloring was associated to the histograms...perhaps a non-blue based color scheme for those would help make that distinction from the other "blues".

Reply:
I've changed it to purple which I think looks a bit more obvious.